projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -47,6 +47,12 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
   })
   public additionalChildSpecifications: Specification[] = [];
 
+  @ModelProperty({
+    key: 'filters',
```

Comment:
Filters already exists on the base gql data source model which this extends, and should be included in the filters passed in on line 61 (although it's possible there's a bug of course)